src/allotropy/parsers/appbio_quantstudio_designandanalysis/structure/generic/structure.py

```diff
@@ -151,11 +157,21 @@ def create(
             f"Unable to find well position for Well '{identifier}'.",
         ]
 
-        amp_data = contents.get_non_empty_sheet("Amplification Data")
-        melt_curve_data = contents.get_non_empty_sheet_or_none("Melt Curve Raw")
+        amp_data = cls.get_amplification_data_sheet(contents)
+        amplification_data = (
+            AmplificationData.create(amp_data, identifier, target_dna_description)
+            if amp_data is not None
+            else None
```

Comment:
Just flagging in case @nathan-stender 's comment above applies here https://github.com/Benchling-Open-Source/allotropy/pull/584/files#r1718937680

Reply:
probably not since this is not a quantity 🤦🏽

Reply:
Yup that util is just a special case for the very common check when setting TQuantityValues in ASM